scripts/compare_changelog_commits.sh

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+
+# This script compares the list of merged PRs in Git with the list of PRs in the CHANGELOG.
+# It prints the authors of the merged PRs and the PRs in the CHANGELOG, and highlights any discrepancies.
+
+# Last release date
+LAST_RELEASE_DATE="2025-05-05"
+
+# Path to ChangeLog
+CHANGELOG_FILE="../.github/CHANGELOG.md"
+
+# -------------------------------------------------------------------------------------------------
+# Script body
+# -------------------------------------------------------------------------------------------------
+
+# Test if CHANGELOG file exists
+if [ ! -f "$CHANGELOG_FILE" ]; then
+    echo "CHANGELOG file not found at $CHANGELOG_FILE"
+    exit 1
+fi
+
+# Check if gh CLI is installed
+if ! command -v gh &> /dev/null; then
+    echo "gh CLI is not installed. Please install it to run this script."
+    exit 1
+fi
+
+# Find the end of the current version section in the CHANGELOG
+changelog_lower_bound=$(grep -n -- "---" $CHANGELOG_FILE | head -n 1 | cut -d: -f1)
+
+# Find the beginning of the Contributors section
+contributors_begin=$(grep -n "<h3>Contributors ✍️</h3>" $CHANGELOG_FILE | head -n 1 | cut -d: -f1)
+
+# Print list of Authors from Changelog and Git
+echo "Authors in CHANGELOG       /  Authors in Git"
+echo "---------------------------|---------------------------"
+paste <(sed -n "$((contributors_begin + 4)),$((changelog_lower_bound - 2))p" $CHANGELOG_FILE | sort | sed 's/,//; s/$/,/') <(git log master --since=$LAST_RELEASE_DATE | grep "Author:" | sort | uniq | sed 's|<.*|| ; s|Author: ||' ) | column -t -s ','
```

Comment:
```suggestion
paste <(sed -n "$((contributors_begin + 4)),$((changelog_lower_bound - 2))p" "${CHANGELOG_FILE}" | sort | sed 's/,//; s/$/,/') <(git log master --since=$LAST_RELEASE_DATE | grep "Author:" | sort | uniq | sed 's|<.*|| ; s|Author: ||' ) | column -t -s ','
```